lib/Migration/Version010003Date20231208083112.php

```diff
@@ -0,0 +1,133 @@
+<?php
+// SPDX-FileCopyrightText: Sami Finnilä <sami.finnila@nextcloud.com>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+declare(strict_types=1);
+
+namespace OCA\TPAssistant\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version010003Date20231208083112 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return void
+	 */
+	public function preSchemaChange(IOutput $output, Closure $schemaClosure, array $options): void {
+	}
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options) {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if (!$schema->hasTable('assistant_i_prompts')) {
+			$table = $schema->createTable('assistant_i_prompts');
+			$table->addColumn('id', Types::BIGINT, [
+				'autoincrement' => true,
+				'notnull' => true,
+			]);
+			$table->addColumn('user_id', Types::STRING, [
+				'notnull' => true,
+				'length' => 64,
+			]);
+			$table->addColumn('value', Types::STRING, [
+				'notnull' => true,
+				'length' => 1000,
+			]);
+			$table->addColumn('timestamp', Types::BIGINT, [
+				'notnull' => true,
```

Comment:
```suggestion
				'notnull' => true,
				'unsigned' => true,
```
For all the timestamps.